server/users.js

```diff
@@ -1,31 +1,123 @@
 'use strict'
 
 const db = require('APP/db')
-const User = db.model('users')
+const User = db.model('users'),
+  Address = db.model('addresses'),
+  Order = db.model('orders'),
+  Review = db.model('reviews')
 
 const {mustBeLoggedIn, forbidden} = require('./auth.filters')
+const router = require('express').Router()
 
-module.exports = require('express').Router()
-  .get('/',
-    // The forbidden middleware will fail *all* requests to list users.
-    // Remove it if you want to allow anyone to list all users on the site.
-    //
-    // If you want to only let admins list all the users, then you'll
-    // have to add a role column to the users table to support
-    // the concept of admin users.
-    forbidden('listing users is not allowed'),
-    (req, res, next) =>
-      User.findAll()
-        .then(users => res.json(users))
-        .catch(next))
-  .post('/',
-    (req, res, next) =>
-      User.create(req.body)
-      .then(user => res.status(201).json(user))
-      .catch(next))
-  .get('/:id',
-    mustBeLoggedIn,
-    (req, res, next) =>
-      User.findById(req.params.id)
-      .then(user => res.json(user))
-      .catch(next))
+module.exports = router
+
+router.param(':userId', (req, res, next, userId) => {
+  User.find({
+    where: {id: userId},
+    include: [
+      {model: Address, as: 'shippingAddresses'},
```

Comment:
you include Address in your default scope, I believe -- no reason to eager load them here